vector/src/main/res/values/strings.xml

```diff
@@ -322,6 +322,14 @@
     <string name="start_chatting">Start Chatting</string>
     <string name="spaces">Spaces</string>
 
+    <!-- Time units -->
+    <plurals name="time_unit_hour_short">
+        <item quantity="one">hour</item>
+        <item quantity="other">hours</item>
+    </plurals>
+    <string name="time_unit_minute_short">min</string>
+    <string name="time_unit_second_short">sec</string>
```

Comment:
It's weird to have a plural for hours, but not for minutes and seconds. It can be better to have only plurals to give more flexibility to the translators.
All plural items for English can be identical.

Reply:
I agree. I decided to remove the plural and use "h" instead of "hour" to be consistent with other platforms. See [e9d9319](https://github.com/vector-im/element-android/pull/5989/commits/e9d93194f15f58c3ce10e132b6fc244de4c8f5bb)